packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -62,8 +62,10 @@ const AccountTransferReceipt = ({
     };
 
     const switchAndRedirect = async () => {
-        await switchAccount(switch_to.value);
-        openStatement();
+        if (switch_to.value) {
+            await switchAccount(switch_to.value);
+            openStatement();
+        }
```

Comment:
`switchAccount` does accept a `string`, So I had to add a check for a non-null value.